data/test/scenarios/ability_id_not_active_test.cfg

```diff
@@ -0,0 +1,86 @@
+#####
+# API(s) being tested: [have_unit]ability_id_active
+##
+# Actions:
+# Give Alice a [hides] ability active only on forests.
+# Recruit a Peasant and check that no unit exists with an active [hides] ability.
+# Recruit a Woodsman and check that no unit exists with an active [hides] ability.
+##
+# Expected end state:
+# [not][have_unit]ability_id_active does not trigger on either recruit
+#####
+
+# wmllint: no translatables
```

Comment:
I think this line should be the first line of the file, because the `#textdomain` should be the first line of any file with translatables.